chart/templates/service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: poc-acapy-multitenant
+  labels:
+    app: poc-acapy-multitenant
+    chart: poc-acapy-multitenant-0.1.0
+    release: poc-acapy-multitenant
+    heritage: Helm
+spec:
+  type: ClusterIP
+  ports:
+    - port: 9836
```

Comment:
on the instructions posted

PORTS="8040:8040 8041:8041"

which ones are the correct now ?

Reply:
updated

Reply:
this are correct PORTS="8040:8040 8041:8041"